doc/command_ref.rst

```diff
@@ -241,7 +241,7 @@ Check Update Command
 -------------
 Clean Command
 -------------
-Performs cleanup of temporary files for the currently enabled repositories.
+Performs cleanup of temporary files.
```

Comment:
explicitly write that this command ignores `--enablerepo, --disablerepo, --releasever`